<ns0:entry xmlns:ns0="http://www.w3.org/2005/Atom" xmlns:ns1="http://schemas.google.com/g/2005" xmlns:ns2="http://schemas.google.com/projecthosting/issues/2009" ns1:etag="W/&quot;DEQAQn47eCl7ImA9Wx9XEUg.&quot;">
		<ns0:id>http://code.google.com/feeds/issues/p/nativeclient/issues/1285/comments/full/1</ns0:id><ns0:author>
			<ns0:name>dmichael@google.com</ns0:name><ns0:uri>/u/dmichael@google.com/</ns0:uri></ns0:author><ns0:content type="html">This bug can be reproduced pretty easily on Windows as follows:
   &gt; mkdir test
   &gt; mkdir test\1
   &gt; mklink /D test\current test\1
   &gt; python -c "import chromium_utils; chromium_utils.RemoveDirectory('test')"

I tried fixing by adding a simple check with os.path.islink in this CL:
http://codereview.chromium.org/5975009/

But it turns out that os.path.islink() doesn't work in Python 2.6 on Windows.  It's only fixed in recent versions of Python 3.  See:
http://bugs.python.org/issue1578269

So this requires a somewhat ugly workaround using shutil.rmtree's error handling.  CL coming soon.</ns0:content><ns0:updated>2011-01-04T15:52:23.000Z</ns0:updated><ns0:published>2011-01-04T15:52:23.000Z</ns0:published><ns2:updates /><ns0:title>Comment 1 by dmichael@google.com</ns0:title><ns0:link href="http://code.google.com/p/nativeclient/issues/detail?id=1285#c1" rel="alternate" type="text/html" /><ns0:link href="https://code.google.com/feeds/issues/p/nativeclient/issues/1285/comments/full/1" rel="self" type="application/atom+xml" /></ns0:entry>